From: Shengjing Zhu <i@zhsj.me>
Date: Sat, 15 Apr 2017 10:50:14 +0000
Subject: disable tests

---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 0cf8ff5..8deb5e1 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -73,4 +73,4 @@ add_subdirectory(doc)
 add_subdirectory(data)
 add_subdirectory(include)
 add_subdirectory(src)
-add_subdirectory(tests)
+#add_subdirectory(tests)
